RoboJackets/urc-software
# Feat/motor controller

# Description

This PR does the following:
- Adds a motor controller node to `urc_platform`
- Adds Ethernet socket support to the `urc_util` package
- Fixes `nanopb` relative pathing issues by correctly binding necessary header files to `urc_nanopb`
- Adds a *skeleton* for unit testing (need to wait for ports to be settled by electrical)

Fixes #8.

# Testing steps (If relevant)
1. Make sure that the `nanopb` is correctly added by running `git submodule update --init`
2. Run `colcon build`

Expectation: Build without errors, specifically from the modified packages.

# Self Checklist
